starter-code/iteration2/src/app/main/main.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, OnInit } from "@angular/core";
+
+@Component({
+  selector: "app-main",
+  templateUrl: "./main.component.html",
+  styleUrls: ["./main.component.css"]
+})
+export class MainComponent implements OnInit {
+  main: object = new Main(
```

Comment:
Since this is an instance of `Main` class, it would be more correct to declare it as type Main:
`main: Main = new Main(...`

Some people prefer not to declare a type at all for a `new` statement, since it is already obvious what type it is:
`main = new Main(...`